mempool/clist_mempool.go

```diff
@@ -648,41 +712,49 @@ func (mem *CListMempool) recheckTxs() {
 		return
 	}
 
-	mem.recheck.init(mem.txs.Front(), mem.txs.Back())
+	// Recheck all transactions in each lane, sequentially.
+	// TODO: parallelize rechecking on lanes?
```

Comment:
I guess a PR/issue to handle this TODO is on its way. I don't think we should leave this in this state, even for an MVP

Reply:
It was easier to recheck lane by lane sequentially. Doing it in parallel will require some more work. I agree to not leave the TODO here.